chemper/graphs/cluster_graph.py

```diff
@@ -362,16 +362,16 @@ def __init__(self, mols=None, smirks_atoms_lists=None, layers=0):
         ----------
         mols: list of chemper Mols
             (optional) molecules to initiate ClusterGraph
-        smirks_atoms_lists: list of list of dict
+        smirks_atoms_lists: list of list of tuples
             (optional) atom indices by smirks index for each molecule
             required if a list of molecules is provided.
-            This is a list of dictionaries of the form [{smirks index: atom index}]
-            for each molecule provided
-            # TODO: not now, but I would like to rework this to just use a tuple of
-            # atom indices instead of knowing the SMIRKS, where the smirks index
-            # would be tuple index + 1 so it would be a list of list of tuples where
-            # [ [ (0,1) ] ] would indicate that for molecule 0 smirks atom 1 label go to
-            # atom with index 0 and smirks atom 2 label go to atom with index 1
+            For each molecule there should be a list of tuples.
+            Each tuple specifies atom indices which should be
+            used to generate indexed SMIRKS atoms, that is they have a `:n` at the end
+
+            For example, if you want to specify atoms 0 and 1 in two molecules this list
+            would look like
+            [ [ (0,1) ], [ (0,1) ] ]
```

Comment:
@vtlim to answer this question. You imagine wanting multiple bonds in the same molecule to be assigned the same same parameter. So that is intended to be a list of tuples for each molecule where the outer list has the same length as the molecules list. Do you think there is a better way ask for this input data. 

I'm not attached to the API so we could make this one big input with 
```
[ (molecule, [list of tuples] ), 

.... 
]
```
with one of these big tuples for each molecule.